dist/templates/ovnkube-single-node-zone.yaml.j2

```diff
@@ -236,132 +236,11 @@ spec:
-        - name: OVNKUBE_CONFIG_DURATION_ENABLE
-          value: "{{ ovnkube_config_duration_enable }}"
-        - name: OVN_NET_CIDR
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: net_cidr
-        - name: OVN_SVC_CIDR
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: svc_cidr
-        - name: K8S_APISERVER
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: k8s_apiserver
-        - name: K8S_NODE
-          valueFrom:
-            fieldRef:
-              fieldPath: spec.nodeName
-        - name: K8S_NODE_IP
-          valueFrom:
-            fieldRef:
-              fieldPath: status.hostIP
-        - name: OVN_KUBERNETES_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: OVN_HYBRID_OVERLAY_ENABLE
-          value: "{{ ovn_hybrid_overlay_enable }}"
-        - name: OVN_ADMIN_NETWORK_POLICY_ENABLE
-          value: "{{ ovn_admin_network_policy_enable }}"
-        - name: OVN_EGRESSIP_ENABLE
-          value: "{{ ovn_egress_ip_enable }}"
-        - name: OVN_EGRESSSERVICE_ENABLE
-          value: "{{ ovn_egress_service_enable }}"
-        - name: OVN_EGRESSIP_HEALTHCHECK_PORT
-          value: "{{ ovn_egress_ip_healthcheck_port }}"
-        - name: OVN_EGRESSFIREWALL_ENABLE
-          value: "{{ ovn_egress_firewall_enable }}"
-        - name: OVN_EGRESSQOS_ENABLE
-          value: "{{ ovn_egress_qos_enable }}"
-        - name: OVN_MULTI_NETWORK_ENABLE
-          value: "{{ ovn_multi_network_enable }}"
-        - name: OVN_HYBRID_OVERLAY_NET_CIDR
-          value: "{{ ovn_hybrid_overlay_net_cidr }}"
-        - name: OVN_DISABLE_SNAT_MULTIPLE_GWS
-          value: "{{ ovn_disable_snat_multiple_gws }}"
-        - name: OVN_EMPTY_LB_EVENTS
-          value: "{{ ovn_empty_lb_events }}"
-        - name: OVN_V4_JOIN_SUBNET
-          value: "{{ ovn_v4_join_subnet }}"
-        - name: OVN_V6_JOIN_SUBNET
-          value: "{{ ovn_v6_join_subnet }}"
-        - name: OVN_SSL_ENABLE
-          value: "{{ ovn_ssl_en }}"
-        - name: OVN_GATEWAY_MODE
-          value: "{{ ovn_gateway_mode }}"
-        - name: OVN_MULTICAST_ENABLE
-          value: "{{ ovn_multicast_enable }}"
-        - name: OVN_ACL_LOGGING_RATE_LIMIT
-          value: "{{ ovn_acl_logging_rate_limit }}"
-        - name: OVN_ENABLE_INTERCONNECT
-          value: "{{ ovn_enable_interconnect }}"
-        - name: OVN_ENABLE_MULTI_EXTERNAL_GATEWAY
-          value: "{{ ovn_enable_multi_external_gateway }}"
-        - name: OVN_HOST_NETWORK_NAMESPACE
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: host_network_namespace
-        - name: OVN_NORTH
-          value: "local"
-        - name: OVN_SOUTH
-          value: "local"
-      # end of container
-
       - name: ovnkube-node
```

Comment:
let's call this ovnkube-controller for parity sake with ovn-controller

Reply:
so when we join ovnkube-node with ovnkube-controller, maybe we need a new name for it? 
I chose ovnkube-node container name, since it is the main part of ovnkube-node pod, but we can consider other options

Reply:
ovnkube-node-controller? :rofl:

Reply:
all the slides i have and sent out say ovnkube-controller, its already rolling off well, i hope we don't do another change again.. ovnkube-controller should rhyme with ovn-controller...